app/models/manageiq/providers/vmware/infra_manager/vm/reconfigure.rb

```diff
@@ -357,6 +364,27 @@ def remove_network_adapter_config_spec(vim_obj, vmcs, hardware, options)
     end
   end
 
+  def edit_network_adapter_config_spec(vim_obj, vmcs, hardware, options)
+    network_name = options[:name]
+    network_vlan = options[:network]
+
+    _log.info("Change network #{network_name} to portGroup #{network_vlan}")
+    device = vim_obj.getDeviceByLabel(network_name, hardware)
+    raise "edit_network_adapter_config_spec: Network device with name #{network_name} was not found." if device.nil?
+
+    lan = Lan.find_by(:name => network_vlan, :switch_id => HostSwitch.where(:host_id => host).pluck(:switch_id))
+    raise "edit_network_adapter_config_spec: network name '#{network_vlan}' not found" unless lan
+
+    device.backing.port = VimHash.new("DistributedVirtualSwitchPortConnection") do |dev|
```

Comment:
This doesn't take into consideration if the network is a standard network or a distributed virtual portgroup

Reply:
Think you just need to do what you have here https://github.com/ManageIQ/manageiq-providers-vmware/blob/678193151a4b1040fadadd6a06d2574571cac339/app/models/manageiq/providers/vmware/infra_manager/vm/reconfigure.rb#L296-L306

Reply:
Yep updated and tested. Local netwerk also work now